test/libsolidity/smtCheckerTests/inline_assembly/assembly_local_storage_access_inside_function.sol

```diff
@@ -17,8 +15,10 @@ contract C {
 		assert(i == 32); // should hold, not changed by the assembly
 	}
 }
+// ====
+// SMTEngine: all
 // ----
-// Warning 7737: (116-182): Inline assembly may cause SMTChecker to produce spurious warnings (false positives).
-// Warning 6328: (185-200): CHC: Assertion violation happens here.\nCounterexample:\nz = 0\ni = 32\n\nTransaction trace:\nC.constructor()\nState: z = 0\nC.f()
-// Warning 6328: (219-233): CHC: Assertion violation happens here.\nCounterexample:\nz = 43\ni = 32\n\nTransaction trace:\nC.constructor()\nState: z = 0\nC.f()
-// Warning 7737: (116-182): Inline assembly may cause SMTChecker to produce spurious warnings (false positives).
+// Warning 7737: (83-149): Inline assembly may cause SMTChecker to produce spurious warnings (false positives).
+// Warning 6328: (152-167): CHC: Assertion violation happens here.\nCounterexample:\nz = 0\ni = 32\n\nTransaction trace:\nC.constructor()\nState: z = 0\nC.f()
+// Warning 6328: (186-200): CHC: Assertion violation happens here.\nCounterexample:\nz = 0\ni = 32\n\nTransaction trace:\nC.constructor()\nState: z = 0\nC.f()
+// Warning 7737: (83-149): Inline assembly may cause SMTChecker to produce spurious warnings (false positives).
```

Comment:
Is this correct? `z` changed in the counterexample but not in the trace. There are a few other changes in this PR that look similar (e.g. `modifier_same_local_variables.sol `) but they change both the counterexample and the trace.

Reply:
The `State: z = 0` in the trace shows `z`'s value after the constructor, whereas `Counterexample: z = 0` shows `z`'s value when the assertion failed. Both 43 and 0 are valid cases there, because of `sstore`, and the solver just chose a number that breaks the assertion. So this looks fine to me.

Reply:
ok. great.